tests/test_StellarLocus.py

```diff
@@ -69,7 +70,9 @@ def test_wPerp(self):
         config = WPerpTask.ConfigClass()
         task = WPerpTask(config=config)
         result = task.calcWPerp('wPerp', cat, ext_vals)
-        self.assertEqual(result.measurement.quantity, expected_wperp)
+        self.assertEqual(result.measurement.quantity.unit, expected_wperp.unit)
```

Comment:
I thought that the measurement.quantity was doing a "close" comparison behind the scenes (though that's not obvious!) but at the same time there's no control over the precision of the match, so this change is probably necessary.